cpp/dolfinx/refinement/plaza.cpp

```diff
@@ -103,12 +103,9 @@ void enforce_rules(
 //-----------------------------------------------------------------------------
 // 2D version of subdivision allowing for uniform subdivision (flag)
 std::vector<std::int32_t>
-get_triangles(const std::vector<std::int8_t>& marked_edges,
+get_triangles(const std::vector<std::int64_t>& indices,
```

Comment:
As we only ever access the `indices[3:6]` I think we should send in a span instead of the full array. 
If not, we need to clarify that the whole array does not contain edge data.